data/json/monsterdrops/feral_humans.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "type": "item_group",
+    "subtype": "collection",
+    "id": "feral_humans_death_drops_axe",
+    "entries": [ { "item": "fire_axe", "prob": 100, "damage": [ 2, 4 ] }, { "group": "default_zombie_clothes", "prob": 100 } ]
```

Comment:
It looks like it should be `fire_ax`.
https://github.com/CleverRaven/Cataclysm-DDA/blob/12ea5d237de0a4ace996bc8a857eee47a0848edf/data/json/items/tool/firefighting.json#L55-L58